training/src/app/modules/chuck-norris/components/joke-list/joke-list.component.scss

```diff
@@ -0,0 +1,20 @@
+@import "../../../../../styles";
+
+table {
+  width: 78%;
```

Comment:
why 78? why not 79? or 48?  if there is a reason save this as a const.
in this case i dont see a reason, according to Figma the table is 100% of the page